docs/docs/guides/getting_started/getting_started.md

```diff
@@ -6,9 +6,13 @@ sidebar_label: Quickstart
 
 # Quickstart
 
+## Live code editor
+
+<iframe width="100%" height="700px"  src="https://stackblitz.com/edit/vitejs-vite-aksddx?embed=1&file=main.js&showSidebar=1"></iframe>
```

Comment:
this PR is moving stackblitz at top instead of https://github.com/web3/web3.js/issues/6866